pkg/minikube/cluster/cluster_darwin.go

```diff
@@ -66,8 +66,6 @@ func createXhyveHost(config MachineConfig) *xhyveDriver {
 		Boot2DockerURL: config.Downloader.GetISOFileURI(config.MinikubeISO),
 		BootCmd:        "loglevel=3 user=docker console=ttyS0 console=tty0 noembed nomodeset norestore waitusb=10 base host=" + cfg.GetMachineName(),
 		DiskSize:       int64(config.DiskSize),
-		Virtio9p:       true,
-		Virtio9pFolder: "/Users",
 		QCow2:          false,
```

Comment:
I think enabling `minikube mount` by default and adding the daemon should be separate PRs